docs/characters.md

```diff
@@ -952,7 +952,7 @@ The character global unique identifier. This number must be unique and is the be
 
 ### account
 
-The account ID in which this character resides. See [account.id](account_2130004.html#account-id). in the auth database.
+The account ID in which this character resides. See [account.id](http://www.azerothcore.org/wiki/account#id). in the auth database.
```

Comment:
not relative link

Reply:
it's not a big issue, we can bulk search and replace if needed

Reply:
what would be the right link tho?